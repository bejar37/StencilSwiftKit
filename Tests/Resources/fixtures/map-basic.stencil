// map items1
{% map items1 with item set mapped1 %}
	- {{item}}
{% endmap %}

// map items2
{% map items2 with item set mapped2 %}"{{item}}", {% endmap %}

Value of mapped1:
{% for item in mapped1 %}
{{item}}
{% endfor %}

Value of mapped2:
{% for item in mapped2 %}{{item}}{% endfor %}
