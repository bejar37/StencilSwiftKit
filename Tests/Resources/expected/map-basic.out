// map items1

// map items2

Value of mapped1:
	- one
	- two
	- three

Value of mapped2:
"hello", "world", "everyone", 
